### DISCUSSION TITLE
Escalation to the hive from kibana

### DISCUSSION THREAD

**Discussion by sinnetll19** [Security-Onion-Solutions/securityonion#6742] [Unsupported Versions]
when using the push to the hive from kibana i get a 404 page not found error. i am able to escalate to the hive from the SOC with the same user account. i checked that the API key in the global.sls matches the user account on the hive. im using SO 2.3.70. Any help fixing this issue would be great.

**Comment by dougburks** [comment]
Escalating from Kibana to TheHive does not work right now. Also note that licensing for TheHive has changed and so we are building our own case management solution that will be native to SOC. Stay tuned!
